### PR TITLE
[Main Course] - PayPal crawler

### LINKED ISSUE #21: [Main Course] - Job crawler of PayPal
# Outline

- 分析网页结构
- 找出定位元素
- 爬取数据
- 整理数据

# Milestone
- 7/19 - Outline complete
- 7/23 - Draft complete
- 7/23 - Review complete
- 7/26 - Release

### PR DESCRIPTION
**Pull request by BulletTech2021**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #21  (PayPal crawler)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+```
+
+## 3.3 解析职位类别
+
+```Python
+# 找到页面中需要的元素，存储职位列表信息
+def job_parser(html):
+    header,desc,link = [[] for i in range(3)]
+    soup = BeautifulSoup(html, "html.parser")
+    # 右键通过打开留言器检查器，在元素tab中查看网页源码，可看到职位名称的类别名字为primary-text-color job-result-title，并且是一个a标签
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md:76]: 浏览器检查器

File: docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-7-19/1626698254470-JD.png" width="700" />
+  <figcaption>PayPal职位描述</figcaption>
+</figure>
+
+## 3 用代码构建爬虫
+
+Python环境的配置请参考之前的文章：[两分钟打造淘宝抢单机器人](https://mp.weixin.qq.com/s?__biz=MzI4Mjk3NzgxOQ==&mid=2247484044&idx=1&sn=ed68ae67ea2f1360053e4ecae0be3ba7&chksm=eb90f1f8dce778eeba987ea0b2f37341999418fd9541c4c7a7b1c256dbada1185e6f4c2aae87&token=261686941&lang=zh_CN#rd)
+
+## 3.1 导入依赖的包
```
Comment (jhqian0909) [inline docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md:40]: 感觉可以用###

File: docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md (hunk 1 above)
Comment (BulletTech2021) [reply docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md]: Fixed

File: docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md (hunk 2 above)
Comment (BulletTech2021) [reply docs/Main_Course/Data_Analytics/2021-07-18-PayPal-Job-Crawler.md]: fixed

**Review by jhqian0909 — APPROVED** [review]

**Pull request merged**